Snake/BusinessLayer/AI.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+
+namespace BusinessLayer
+{
+    public class AI
+    {
+        private Direction currentDirection;
+        private int X;
+        private int Y;
+        Random randomNumber = new Random();
+        public int XCoordinate
+        {
+            get
+            {
+                return X;
+            }
+            set

```

Comment:
Do we ever want anything outside of this class to be able to change toe X or Y coordinates? If not you can remove the set.